lib/Crypto/DecryptAll.php

```diff
@@ -149,12 +171,12 @@ public function prepare(InputInterface $input, OutputInterface $output, $user) {
 		$privateKey = $this->getPrivateKey($user, $password);
 		if ($privateKey !== false) {
 			$this->updateSession($user, $privateKey);
+			$output->writeLn('Setup encryption module ready for decryption');
```

Comment:
this looks more like debugging information, not sure we should show this to an admin...

Reply:
Removed as of now.